web/src/components/TraceDrawer/useChevronDirectionMemo.tsx

```diff
@@ -0,0 +1,13 @@
+import {useMemo} from 'react';
+
+export function useChevronDirectionMemo(height: number | undefined, max: number | undefined, min: number | undefined) {
```

Comment:
You can give a default to the optional parameters by doing something like:
```
export function useChevronDirectionMemo(height: = 0, max: = 0, min:  =0) {
```

This way you are saying both that the type is `number` and if undefined the default value should be `0`